assets/src/scripts/monitoring-location/components/hydrograph/graph-brush.js

```diff
@@ -12,19 +12,32 @@ import {getBrushLayout} from './selectors/layout';
 import {getBrushXScale, getBrushYScale} from './selectors/scales';
 import {isVisible} from './selectors/time-series-data';
 import {drawDataLines} from './time-series-lines';
+import {mediaQuery} from '../../../utils';
+import config from '../../../config';
 
 export const drawGraphBrush = function(container, store) {
+    let customHandle;
+    let layoutHeight;
 
     const brushed = function() {
+        // if the user clicks a point in the brush area without making an actual selection, remove the custom handles
+        if (event.selection == null) {
+            customHandle.attr('display', 'none');            
+        }
+        customHandle.attr('transform', function(d, index) {
+            const yPositionForCustomHandle = mediaQuery(config.USWDS_LARGE_SCREEN) ? layoutHeight / 3.3 * -1 : layoutHeight / 2.5 * -1;
```

Comment:
Assuming that 3.3 and 2.5 are just "magic" constants. Rather than multipling by -1 can we just apply the negative sign to layoutHeight?

Reply:
lol. Why didn't I think of that?? Also will add the magic numbers as constants with names that might make their function clearer.